Templates/index.md

```diff
@@ -1,32 +1,36 @@
+| Template | Description|
+| --- | ---|
+|[Overview](../Templates/Feature-Overview-Template.md)| Provides a quick outline of a feature. Links to more detailed information are provided at the bottom of the document.|
+|[Explainer](../Templates/Feature-Explainer-Template.md)|Provides a guideline to use when documenting a feature. It provides information about the feature, how it fits into the big picture of the EOS blockchain as well as any dependencies and constraints. It also provides information on related use cases and expected results, as well as example code.|
+|[Tutorial](../Templates/Tutorial-Template.md)| Provides an educational lesson with context. It contains one or more procedures to demonstrate mastery of a concept. A tutorial instructs users how to perform/achieve a particular goal. You need to add test data or tools necessary to complete and test the tutorial. A tutorial should contain 7 or less steps, if possible. If you have more than 7 steps, consider dividing the tutorial into two separate tutorials. Keep the tutorial experience for the user to an hour or less, if possible.|     
+|[How To](../Templates/How-to-Template.md)| This template provides step-by-step instructions to produce a desired result. It provides prerequisite information, when to use it,  as well as expected results. This template also provides an option for example code.|
 
-      c. [Tutorial](../Templates/Tutorial-Template.md): This template provides an educational lesson with context. It contains one or more procedures to demonstrate mastery of a concept. A tutorial instructs users how to perform/achieve a particular goal. You need to add test data or tools necessary to complete and test the tutorial. A tutorial should contain 7 or less steps, if possible. If you have more than 7 steps, consider dividing the tutorial into two separate tutorials. Keep the tutorial experience for the user to an hour or less, if possible.
-      
-      d. [How To](../Templates/How-to-Template.md): This template provides step-by-step instructions to produce a desired result. It provides prerequisite information, when to use it,  as well as expected results. This template also provides an option for example code.
-4. Copy the contents of the template file
-5. Paste the contents of the template file into your new document and then write your content
+## How to Apply a Template to Your Content
+Complete the following steps to apply a template to your content:
+1. Open a new document in your .md editor.
+2. Open the DevRel repository [Templates](../Templates/index.md) folderthen select a template from the [Available Templates](#available-templates).
+3. Copy the contents of the template file.
+4. Paste the contents of the template file into your new document and then write your content.
```

Comment:
needs the "and"

Reply:
I think comma or and are syntactically fine. When listing multiple actions in the same step the comma is preferred since it's more clear for the reader how many actions there are. The and doesn't provide that. @VEG-Ginna